docker-run-opencv-ovms.sh

```diff
@@ -5,6 +5,12 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
+SERVER_CONTAINER_NAME="model-server"
+CLIENT_CONTAINER_NAME_PREFIX="ovms-client"
+# clean up exited containers
+docker rm $(docker ps -a -f name=$SERVER_CONTAINER_NAME -f status=exited -q)
```

Comment:
if you do this, please also do `removing --rm` from server container

Reply:
it's done and included in last commit